docs/contributing.md

```diff
@@ -1,26 +1,34 @@
-# How to Contribute to PeopleDepot
```

Comment:
Can we keep this?

Reply:
Yes, please?

"Overview" or "Welcome to PeopleDepot Contributing Guideline" works for me, too.